change/@fluentui-react-positioning-2c2ce771-85d7-4485-9269-448fac6cdda2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32092&commitId=a0e5790d8e997ff297c91cc5e9f4bff3c679d0e8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|2| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|29| Changed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|218| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|6| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|6| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|461| Changed|
|Positioning.arrow.chromium.png|486| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|130| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|8| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|8| Changed|
|Tooltip Converged.withArrow.chromium.png|1| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|7| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|1| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
There are _subpixel_ differences, but they are expected as the implementation changed. Anyway, the whole thing looks more correct now.